demo/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <application
+        android:label="@string/app_name"
```

Comment:
Do we want to declare: `android:networkSecurityConfig` or `android:usesCleartextTraffic` ?

Reply:
I don't think so. I think we want to, by default, require SSL. The app integrating the SDK can then decide how to configure to talk to their specific API server.